scripts/tools/check_includes_config.py

```diff
@@ -136,6 +136,7 @@
     'src/app/clusters/media-input-server/media-input-delegate.h': {'list'},
     'src/app/clusters/media-playback-server/media-playback-delegate.h': {'list'},
     'src/app/clusters/target-navigator-server/target-navigator-delegate.h': {'list'},
+    'src/app/clusters/webrtc-requestor-server/webrtc-requestor-server.h': {'string', 'vector'},
```

Comment:
The file name does not match the cluster name, if they are not match, they can not be build directly